vividus-plugin-mobile-app/src/main/resources/spring.xml

```diff
@@ -55,6 +55,12 @@
 
     <bean id="iOSPerformanceSteps" class="org.vividus.bdd.mobileapp.steps.IOSPerformanceSteps" />
 
+    <bean id="networkConditionsSteps" class="org.vividus.mobileapp.steps.NetworkConditionsSteps">
+        <constructor-arg>
+            <bean name="actions" class="org.vividus.mobileapp.action.NetworkActions"/>
```

Comment:
actions name does not play any role unless you use @ConstructorProperties or compile the code with debug flag